libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -928,10 +928,15 @@ void AP_GPS::update_instance(uint8_t instance)
                 if (i != instance && _type[i] == GPS_TYPE_UBLOX_RTK_ROVER) {
                     // pass the data to the rover
                     inject_data(i, rtcm_data, rtcm_len);
-                    drivers[instance]->clear_RTCMV3();
                     break;
                 }
             }
+#if GPS_MOVING_BASELINE && !defined(HAL_BUILD_AP_PERIPH)
```

Comment:
why !defined(HAL_BUILD_AP_PERIPH)? This is a perfect Periph application

Reply:
We do it already over CAN. This for MAVLink, but your right some Periph's do have MAVLink so could be `HAL_GCS_ENABLED`.

I mostly had Tracker in mind.